public/app/plugins/panel/piechart/PieChart.tsx

```diff
@@ -30,6 +30,7 @@ import { css } from '@emotion/css';
 import { useComponentInstanceId } from '@grafana/ui/src/utils/useComponetInstanceId';
 import { getTooltipContainerStyles } from '@grafana/ui/src/themes/mixins';
 import { selectors } from '@grafana/e2e-selectors';
+import { darken, lighten } from '@grafana/data/src/themes/colorManipulator';
```

Comment:
```suggestion
import { colorManipulator } from '@grafana/data';
```

and then use it as `colorManipulator.lighten|darken`

When working in the `public/app/plugins/...` it does not really matter, but we should use the standard export paths when possible